Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Qualys_Enrichment.yml

```diff
@@ -1,15 +1,15 @@
 id: Cortex ASM - Qualys Enrichment
 version: -1
 name: Cortex ASM - Qualys Enrichment
-description: Given the IP address this playbook enriches information from Qualys assets.
+escription: Given the IP address this playbook enriches information from Qualys assets.
```

Comment:
```suggestion
description: Given the IP address this playbook enriches information from Qualys assets.
```

Reply:
that should fix this build error: https://app.circleci.com/pipelines/github/demisto/content/295010/workflows/01496a17-45a3-4eb3-881c-1b92afae8b2f/jobs/716722?invite=true#step-119-164

Reply:
resolved in https://github.com/demisto/content/pull/29098/commits/7faeccb53ba3109b146c85d3643ad539a268dcd0